Makefile

```diff
@@ -3,7 +3,7 @@ NAMESPACE=pam
 NAME=oktapam
 BINARY=terraform-provider-${NAME}
 # On version changes, update tag-checks.yml
```

Comment:
@leharbhatt-okta  / @SajanAlexander-okta what does this comment mean?

Reply:
TLDR; you can take out that comment

So I think the idea here was that `tags-checks.yml` would define a GHA which would check out different versions of the provider and run the integration tests.  In theory, this would help us find regressions where a change in platform would cause a regression in an older version of the provider.  The implementation never really got off the ground (i.e. there were actually no old versions defined in `tags-checks.yml`) and the action errored out each time it was run, which resulted in removing the file.

Reply:
Got it. Removed.